### DISCUSSION TITLE
How to reduce Vercel bandwidth on the landing page?

### DISCUSSION THREAD

**Discussion by GorvGoyl** [vercel/community#4861] [Help]
### Summary

My frontend is built using next.JS and deployed on vercel. On my landing page, there is a demo video (above the scroll) and it consumes a lot of bandwidth on a monthly basis. Last month it cost me $80. what are some solutions for this?

PS: Video is already optimized. I can't lazy load it because it's above the scroll.

I'm thinking of using some Cloudflare service for this as they have cheaper rates but not sure how to work with it.

### Steps to Reproduce

1. build a landing page and include a video 
2. host it on vercel
3. make it viral
4. check billing email

**Comment by z4nr34l** [comment]
Use `next/image` for optimizing images (but link do sensible-size images, not full-res 4K if that's really needed).

Use https://next-video.dev/ for video delivery.

Make sure that you are using cache-ing/SSR/SSG on that page.

You can make more information on my 2 posts:

https://www.zanreal.net/blog/turbocharging-performance
https://www.zanreal.net/blog/next-js-going-production-checklist

Which should be able to help you handle any traffic with sensible costs.

Happy coding!
